ui/app/styles/components/empty-state.scss

```diff
@@ -0,0 +1,37 @@
+.empty-state {
+  align-items: center;
+  color: $grey;
+  display: flex;
+  background: $ui-gray-050;
+  justify-content: center;
+  padding: $spacing-xxl $spacing-s;
+}
+
+.empty-state-content {
+  max-width: 320px;
+}
+
+.empty-state-title {
+  color: $grey;
+  font-size: $size-4;
+  font-weight: $font-weight-semibold;
+  line-height: 1.2;
+  margin-bottom: $spacing-xs;
+}
+
+.empty-state-actions {
+  margin-top: $spacing-xs;
+
+  a,
+  .link,
+  a:not(.button):not(.file-delete-button):not(.tag) {
+    color: $blue;
+    font-size: $size-8;
+    font-weight: $font-weight-semibold;
+    text-decoration: none;
+  }
+
+  > * + * {
```

Comment:
Yeeesss! Lobotomised owl! (with one ear!!) 🙌

Reply:
Yeah there was a great list apart article on it a few years back - https://alistapart.com/article/axiomatic-css-and-lobotomized-owls